### ISSUE TITLE
Why dividing L by 20 to calculate Zdepth, in soiltemp function ?

### ISSUE THREAD

**Issue by abdelkrim-bsr** [ilmenichetti/reclim#1]
Why dividing **L** (layer Thickness) by 20 to calculate **Zdepth?** Is it always correct to divide by a constant (20) whatever is the value (10, 20, or 30 cm) of **L**?

`Zdepth=L/20`

https://github.com/ilmenichetti/reclim/blob/f9366fd758f7f814187c623d80ce8cd65e83754a/R/re_crop.R#L127

**Comment by ilmenichetti** [comment]
True, sorry for the rather poor documentation... it is because we always assume we are working on the 0-20 cm interval...
